corehq/form_processor/submission_post.py

```diff
@@ -511,33 +510,3 @@ def notify_submission_error(instance, exception, message):
         notify_exception(request, message, details=details)
     else:
         logging.error(message, exc_info=sys.exc_info(), extra={'details': details})
-
-
-class SubmissionProcessTracker(object):
```

Comment:
Note, I deleted this from here and moved it to `corehq/form_processor/submission_process_tracker.py`.